omod/src/main/java/org/openmrs/module/coreapps/page/controller/conditionlist/AddConditionPageController.java

```diff
@@ -16,10 +16,28 @@
 
 import org.openmrs.ui.framework.page.PageModel;
 import org.springframework.web.bind.annotation.RequestParam;
+import org.openmrs.module.coreapps.CoreAppsConstants;
+import org.openmrs.api.context.Context;
 
 public class AddConditionPageController {
 	
 	public void controller(PageModel model, @RequestParam(value = "returnUrl", required = false) String returnUrl) {
 		model.addAttribute("returnUrl", returnUrl);
+		String criterion = Context.getAdministrationService().getGlobalProperty(CoreAppsConstants.CONDITIONS_TO_BE_DISPLAYED);
+	    String criterion_array[] = criterion.split(",");
```

Comment:
Did you get a chance to look at our naming convention? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions